examples/docker/run.sh

```diff
@@ -0,0 +1,18 @@
+set -x
+set -e
+
+# Start artipie.
+docker run --rm -d --name artipie -it -v $(pwd)/artipie.yaml:/etc/artipie.yml -v $(pwd):/var/artipie -p 8080:80 artipie/artipie:latest
+
+# Wait for container to be ready for new connections.
+sleep 5
+
+# Pull an image from docker hub.
+docker pull ubuntu
+
+# Push the pulled image to artipie
+docker image tag ubuntu localhost:8080/my-docker/myfirstimage
+docker push localhost:8080/my-docker/myfirstimage
```

Comment:
@Sammers21 I think we need to verify `pull` as well. You can test it with `docker image rm localhost:8080/my-docker/myfirstimage` and `docker pull localhost:8080/my-docker/myfirstimage`